conf/messages

```diff
@@ -47,3 +47,10 @@ dashboard.status.haulier.longName=Goods Vehicle Movement Service
 service.state.AVAILABLE=Available
 service.state.UNAVAILABLE=Known issues
 service.state.UNKNOWN=Unknown
+
+planned_work.title=Planned work that will affect GVMS
+planned_work.link=Return to Check GVMS availability page
```

Comment:
Minor: would be good if this key indicated the target of the link (ie. that it was a link to the status page). Could perhaps be: 

`planned_work.service_availability_status_link`